Assets/Mirror/Components/NetworkTransformBase.cs

```diff
@@ -39,6 +40,15 @@ public enum Compression { None, Much, Lots, NoRotation }; // easily understandab
         // This component could be on the player object or any object that has been assigned authority to this client.
         bool isClientWithAuthority => hasAuthority && clientAuthority;
 
+        [Header("Sensitivity")]
+
+        [Tooltip("Changes to the transform must exceed these values to be transmitted on the network.")]
+        public float localPositionSensitivity = .01f;
+        [Tooltip("Changes to the transform must exceed these values to be transmitted on the network.")]
+        public float localEulerAnglesSensitivity = .01f;
+        [Tooltip("Changes to the transform must exceed these values to be transmitted on the network.")]
+        public float localScaleSensitivity = .01f;
```

Comment:
see my comment https://github.com/vis2k/Mirror/issues/1306 before we merge

Reply:
done